libwapiti/src/api.c

```diff
@@ -25,10 +25,13 @@ static raw_t *api_str2raw(char *seq) {
   int size = 32; // Initial number of lines in raw_t
   int cnt = 0;
   char *line;
+  const unsigned int L = strlen(seq)+1;
+  char *tmp_seq = (char*) malloc(sizeof(char) * L);
+  strncpy(tmp_seq, seq, L);
```

Comment:
Just as a nitpick, it would be nice to use the `xstrdup` function from the included wapiti tools (https://github.com/Jekub/Wapiti/blob/master/src/tools.c#L138) , partly to replace these three lines with one, and partly to fail hard if the allocation fails.

Reply:
Nice! It is a very sensible thing to do, I will correct that at once.